packages/common/src/api/tan-query/useFollowers.ts

```diff
@@ -1,42 +1,68 @@
-import { useQuery } from '@tanstack/react-query'
+import { useInfiniteQuery } from '@tanstack/react-query'
 
 import { userMetadataListFromSDK } from '~/adapters/user'
-import { useAppContext } from '~/context/appContext'
+import { useAudiusQueryContext } from '~/audius-query'
 import { ID, Id, OptionalId } from '~/models/Identifiers'
+import { User } from '~/models/User'
 
 import { QUERY_KEYS } from './queryKeys'
 import { Config } from './types'
 import { useCurrentUserId } from './useCurrentUserId'
 
-type GetFollowersArgs = {
+const LIMIT = 15
+
+type UseFollowersArgs = {
   userId: ID | null | undefined
   limit?: number
-  offset?: number
 }
 
 /**
- * Hook to fetch followers for a user with pagination support.
- * NOTE: this is not an infinite query, it only gives you data by your requested limit/offset args
- * If no userId is provided, it will use the current user's ID from the store.
+ * Hook to fetch followers for a user with infinite query support.
+ * This version supports infinite scrolling and maintains the full list of followers.
  */
 export const useFollowers = (
-  { userId, limit = 10, offset = 0 }: GetFollowersArgs,
+  { userId, limit = LIMIT }: UseFollowersArgs,
   options?: Config
 ) => {
-  const { audiusSdk } = useAppContext()
+  const { audiusSdk } = useAudiusQueryContext()
   const { data: currentUserId } = useCurrentUserId()
 
-  return useQuery({
-    queryKey: [QUERY_KEYS.followers, userId, { limit, offset }],
-    enabled: options?.enabled !== false && !!userId,
-    queryFn: async () => {
-      const { data = [] } = await audiusSdk!.full.users.getFollowers({
+  const {
+    data,
+    hasNextPage,
+    isFetchingNextPage,
+    fetchNextPage,
+    isLoading,
+    ...rest
+  } = useInfiniteQuery({
+    queryKey: [QUERY_KEYS.followers, userId, { limit }],
+    enabled: options?.enabled !== false && !!userId && !!audiusSdk,
```

Comment:
nit: can we put this at the bottom with staleTime to match all the other hooks?
also we're calling it config everywhere else, update the arg name if you don't mind